go-controller/pkg/allocator/pod/pod_annotation_test.go

```diff
@@ -124,6 +125,9 @@ func Test_allocatePodAnnotationWithRollback(t *testing.T) {
 		wantReleaseID             bool
 		wantRelasedIDOnRollback   bool
 		wantErr                   bool
+		isSingleStackIPv4         bool
```

Comment:
nit: to add a comment for what these two extra bolleans are doing -> from what I can tell forcing the single stack ness in tests which by default is dualstack?
instead using these two toggles to tell what the stack should be might have been less confusing:
i.e if dualstack then set both to true, if single stack v4 only set 1 of them to true and same for v6 -> then use those values to determine the config.ipv4 and config.ipv6
but i won't hold for this.. its just confusing to maintain in future can be a FUP.

Reply:
Prefer follow-up please